test/views/ImportVATViewSpec.scala

```diff
@@ -0,0 +1,182 @@
+  private final val originalErrorId = "#original-error"
+  private final val amendedErrorId = "#amended-error"
+
+  def underpaymentFormWithValues(originalValue: String, amendedValue: String): Form[UnderpaymentAmount] =
+    formProvider().bind(Map("original" -> originalValue, "amended" -> amendedValue))
+
+  "Rendering the Import VAT page" when {
+
+    "no errors exist" should {
+      val form: Form[UnderpaymentAmount] = formProvider.apply()
+      lazy val view: Html = injectedView(
+        form,
+        Call("GET", controllers.routes.UnderpaymentTypeController.onLoad().toString)
+      )(fakeRequest, messages)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      s"have the correct page title" in {
+        document.title mustBe ImportVATMessages.pageTitle
+      }
+
+      "not render an error summary" in {
+        document.select("div.govuk-error-summary").size mustBe 0
+      }
+
+      "not render an error message against the field" in {
+        document.select("#value-error").size mustBe 0
+      }
+    }
+
+    "an error exists (no value has been specified for original amount)" should {
+      lazy val form: Form[UnderpaymentAmount] = underpaymentFormWithValues(emptyString, fifty)
+      lazy val view: Html = injectedView(
+        form,
+        Call("GET", controllers.routes.UnderpaymentTypeController.onLoad().toString)
+      )(fakeRequest, messages)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      s"have the correct page title" in {
+        document.title mustBe ImportVATMessages.errorPrefix + ImportVATMessages.pageTitle
+      }
+
+      "render an error summary with the correct message " in {
+        elementText(govErrorSummaryListClass) mustBe ImportVATMessages.originalNonEmpty
+      }
+
+      "render an error message against the field" in {
+        elementText(originalErrorId) mustBe ImportVATMessages.errorPrefix + ImportVATMessages.originalNonEmpty
+      }
+    }
+
+    "an error exists (no value has been specified for amended amount)" should {
+      lazy val form: Form[UnderpaymentAmount] = underpaymentFormWithValues(fifty, emptyString)
+      lazy val view: Html = injectedView(
+        form,
+        Call("GET", controllers.routes.UnderpaymentTypeController.onLoad().toString)
+      )(fakeRequest, messages)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      s"have the correct page title" in {
+        document.title mustBe ImportVATMessages.errorPrefix + ImportVATMessages.pageTitle
+      }
+
+      "render an error summary with the correct message " in {
+        elementText(govErrorSummaryListClass) mustBe ImportVATMessages.amendedNonEmpty
+      }
+
+      "render an error message against the field" in {
+        elementText(amendedErrorId) mustBe ImportVATMessages.errorPrefix + ImportVATMessages.amendedNonEmpty
+      }
+    }
+
+    "an error exists (not a numeric value has been specified for amended amount)" should {
```

Comment:
I think you are missing a test for not a numeric value for original amount

Reply:
not numeric value for original amount added